src/files/filesloader.ts

```diff
@@ -84,7 +84,7 @@ export class FilesLoader {
         fileName: string,
     ) {
         return new Promise<Set<string>>((resolve, reject) => {
-            glob(`**/${fileName}`,
+            glob(`${this.configStore.coverageBaseDir}/${fileName}`,
```

Comment:
do we need to consider the workspace folder as apart of this blob or are we hoping the cwd will assist us with that?

Reply:
`cwd` is sufficient in this case as this glob lookup done per workspace folder. It is possible to create combined glob pattern, same as in `coverageservice.ts` but is more intrusive change.
From end user prospective they yield the same result.